core/common/src/main/java/alluxio/extensions/ExtensionsClassLoader.java

```diff
@@ -82,4 +84,14 @@ public ExtensionsClassLoader(URL[] urls, ClassLoader defaultClassLoader) {
       return mDefaultClassloader.loadClass(name, resolve);
     }
   }
+
+  @Override
+  public Enumeration<URL> getResources(String name) throws IOException {
+    Enumeration<URL> resources = super.getResources(name);
```

Comment:
What's the implication of `super.getResources` returning no resources but `mDefaultClassloader` having the requested resource? Is it possible to get the wrong resource if they have resources of the same name?

Reply:
This can be used by `ServiceLoader` for loading service classes. 

Theoretically this does gives more resources comparing to the original implementation. But I think it aligns with the purpose of this class and is similar to what other methods do in this class. 

I did run it through UFS related tests for HDFS and S3 and haven't observe any issue.